CMake/Dependencies/libkvspic-CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ include(ExternalProject)
 # clone repo only
 ExternalProject_Add(libkvspic-download
 	GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-pic.git
-	GIT_TAG           f0f01f0fe88ec78fc34df4e5fdb83d631f2b7ab3
+	GIT_TAG           307214b71bb51b1de7f726bbaade54568bb46151
```

Comment:
Let's wait until PIC `bug-fix` branch PR is merged into `develop` before updating the hash, that PR is still running CI.

Reply:
7b989f464469d52ef00dd9afaff394e563aafa95 <-- that's the commit you need to use, I just merged your changes, the one you cited doesn't exist anymore because it was on the `bug-fix` branch.